### ISSUE TITLE
Just click on the calendar and a new file will be created, which I don't think is a good idea

### ISSUE THREAD

**Issue by aoaim** [srg-kostyrko/obsidian-journal#40]
I like calendar-like journals, but when i click on the calendar, it will soon create a blank file -- even if i do not plan to write somthing down.

I think this will be a good way: when I click, then create a file, that's ok, but when I write down nothing and just quit, this blank file will remove itself.

Thanks a lot.

**Comment by srg-kostyrko** [comment]
Such functionality will be maybe hard to implement
What about having a modal that confirms creating new note?

**Comment by aoaim** [comment]
Yes this will be fine😊

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Jnolcox** [comment]
Add a setting to feature flag it?

**Comment by srg-kostyrko** [comment]
Yes, this will be configurable in settings so you can enable only if needed

**Comment by tobalsan** [comment]
hey @srg-kostyrko 
Love the plugin and I am using it thoroughly. I'm also facing this little inconvenience where I create journal entries by mistake, without anything inside.
Would love to have just a modal asking if I want to confirm the file creation.

**Comment by srg-kostyrko** [comment]
a new setting was added in v2.0.0 to show confirmation modal when creating new note
